theia-extensions/viewer-prototype/src/common/backend-service.ts

```diff
@@ -0,0 +1,8 @@
+import { CancellationToken } from '@theia/core';
+
+export const backendServicePath = '/services/backend-service';
```

Comment:
Can there be a name conflict with other extensions? Make it needs to be more specific.

Reply:
Changed prefix to /services/theia-trace-extension/<service name>, for the existing service also.